### ISSUE TITLE
Error starting app! on All together now section of site 

### ISSUE THREAD

**Issue by mklimstra** [posit-dev/py-shiny-site#218]

**Comment by gadenbuie** [comment]
Thanks for reporting! We're getting this error:

```
ValueError: Can't find a pure Python 3 wheel for 'statsmodels!=0.14.2,>=0.12'.
See: https://pyodide.org/en/stable/usage/faq.html#why-can-t-micropip-find-a-pure-python-wheel-for-a-package
You can use `await micropip.install(..., keep_going=True)` to get a list of all packages with missing wheels.
```

which results from [ridgeplot removing statsmodels==0.14.2](https://github.com/tpvasconcelos/ridgeplot/issues/197), but which we have bundled in the last release of shinylive.

**Comment by wch** [comment]
I filed an issue on pyodide to update statsmodels from 0.14.2 to 0.14.3: https://github.com/pyodide/pyodide/issues/5093

However, I think this will be a nontrivial task because that version of statsmodels wants an updated version of numpy and scipy.

**Comment by agriyakhetarpal** [comment]
Thanks for the cross-link! `statsmodels` 0.14.3 already works, so we just need to merge pyodide/pyodide#5058; I'll do it as soon as possible.

**Comment by gadenbuie** [comment]
The upstream PR was merged

* https://github.com/pyodide/pyodide/pull/5058

**Comment by wch** [comment]
Great! Hopefully there will be a new release of Pyodide soon.

**Comment by agriyakhetarpal** [comment]
Yes, it's quite close. We expect to release 0.27 stable by the middle of October.
